frontend_app_html-document/src/component/HtmlDocument.jsx

```diff
@@ -6,65 +6,67 @@ import DisplayState from './DisplayState.jsx'
 
 const HtmlDocument = props => {
   return (
-    <div className='wsContentHtmlDocument__contentpage__textnote html-document__contentpage__textnote'>
-      {props.isArchived && (
-        <DisplayState
-          msg={props.t('This content is archived')}
-          type='button'
-          icon='archive'
-          btnLabel={props.t('Restore')}
-          onClickBtn={props.onClickRestoreArchived}
-        />
-      )}
+    <div className='html-document__contentpage__left'>
```

Comment:
please rename this class `html-document__contentpage__left__wrapper` and its according css